examples/defcal.qasm

```diff
@@ -1,36 +1,36 @@
 defcalgrammar "openpulse";
 
-defcal x %0 {
-   play drive(%0), gaussian(...);
+defcal x $0 {
+   play drive($0), gaussian(...);
 }
 
-defcal x %1 {
-  play drive(%1), gaussian(...);
+defcal x $1 {
+  play drive($1), gaussian(...);
 }
 
-defcal rz(angle[20]:theta) %q {
-  shift_phase drive(%q), -theta;
+defcal rz(angle[20]:theta) $q {
+  shift_phase drive($q), -theta;
 }
 
-defcal measure %0 -> bit {
+defcal measure $0 -> bit {
   complex[int[24]] iq;
   bit state;
   complex[int[12]] k0[1024] = [i0 + q0*j, i1 + q1*j, i2 + q2*j, ...];
-  play measure(%0), flat_top_gaussian(...);
-  iq = capture acquire(%0), 2048, kernel(k0);
+  play measure($0), flat_top_gaussian(...);
+  iq = capture acquire($0), 2048, kernel(k0);
   return threshold(iq, 1234);
 }
 
-defcal zx90_ix %0, %1 {
-  play drive(%0, "cr1"), flat_top_gaussian(...);  // uses a non-default
-                                                  // frame labeled "cr1"
+defcal zx90_ix $0, $1 {
+  play drive($0, "cr1"), flat_top_gaussian(...);  // uses a non-default
+                                                   // frame labeled "cr1"
```

Comment:
Seems like the extra space mis-aligns the comments?

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/150/commits/7ba67251679c80311bc500d812ce712fff8e1b24